functions/lib/azure-storage-helper.js

```diff
@@ -215,6 +215,32 @@ const azureStorageHelper = {
         logger.error(`${logPrefix}: updatePupilStatus(): ERROR: ${error.message}`)
       }
     })
+  },
+
+  /**
```

Comment:
this feels like the wrong place for this method, as it's not a generic storage helper.  This should really be in the function folder.

Reply:
@GuyHarwood moved to relevant service within pupil status function